src/Configuration/Programmer/Programmer_cmd.cpp

```diff
@@ -19,57 +19,145 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <iostream>
+
 #include "CFGCommon/CFGCommon.h"
 #include "Programmer.h"
 
+void progressCallback(std::string msg) { std::cout << msg; }
```

Comment:
it should be msg.c_str()

windows might not handle this correctly.

Why use std::cout? Why not CFG_POST_MSG?

Reply:
Just to get use to std::cout. Let me change back to CFG_POST_MSG